src/CamelKNodeProvider.ts

```diff
@@ -100,9 +100,12 @@ export class CamelKNodeProvider implements vscode.TreeDataProvider<TreeNode> {
 								this.processIntegrationList(output);
 							}).catch((error) => { 
 								let errMsg : string = error;
-								if (errMsg.toLowerCase().trim().startsWith('error:')) {
-									utils.shareMessage(extension.mainOutputChannel, `Refreshing Apache Camel K Integrations view using kubectl failed. ${error}`);
-									inaccessible = true;
+								if (errMsg) {
+									var errLower =  String.prototype.toLowerCase.apply(errMsg);
```

Comment:
why not directly errMsg.toLowerCase() instead of String.prototype.toLowerCase.apply(errMsg) ?

Reply:
I believe initially I wasn't declaring errMsg explicitly as a string, so I had to play a bit. I'll see if I can simplify.

Reply:
Looking at this again, it has nothing to do with this PR - so I'll spin this and the other comment off into a separate issue to investigate.

Reply:
https://github.com/camel-tooling/vscode-camelk/issues/102